bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -224,6 +224,22 @@
 		<state pattern="%s" readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="obstruction_alarm">
+		<item-type>Contact</item-type>
```

Comment:
The type choice looks strange to me. Shouldn't it be rather a "Switch" type, meaning there was an obstruction (ON)or not (OFF) ?

Reply:
I hat chosen "Contact", because it is read only, but I agree that "Switch" is more intuitive. Changed to "Switch".